docs/src/sass/_controls.scss

```diff
@@ -155,28 +155,48 @@ $color-active: fade-out($color-hover, 0.5);
 }
 
 .VolumeSlider {
-  width: 8px;
-  height: 100px;
   // margin: 10px 0;
 
   &-value {
     position: absolute;
+  }
+
+  &.Direction-horizontal {
```

Comment:
Could we make these more like the `is*` classes? Perhaps `isHorizontal`/`isVertical` or `modHorizontal`/`modVertical`?